nixos/modules/services/security/step-ca.nix

```diff
@@ -60,7 +60,7 @@ in
         '';
       };
       intermediatePasswordFile = lib.mkOption {
-        type = lib.types.path;
+        type = lib.types.pathNotInStore;
```

Comment:
i think this could be a separate PR, to keep the [one pr for a change guideline](https://github.com/NixOS/nixpkgs/tree/master/lib#make-one-pr-for-each-change) of lib.

Reply:
Done!